src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/overgangsstønad/KortInnholdOvergangsstønad.tsx

```diff
@@ -6,10 +6,11 @@ import HistorikkRadIOvergangsstønad from './HistorikkRadIOvergangsstønad';
 
 const Grid = styled.div`
     display: grid;
-    grid-template-columns: repeat(4, max-content);
+    grid-template-columns: repeat(4, minmax(auto, max-content));
```

Comment:
Denne kan du godt snakke om på demo på fredag! For dette kan være lærdom for alle i teamet - at man får litt mer "skalerbar" løsning